app/views/shared/_flash_alerts.js.erb

```diff
@@ -0,0 +1,7 @@
+<% flash.each do |type, messages| %>
+  <% if alert_type?(type) %>
+    <% Array(messages).each do |message| %>
+      ThreeScale.toast('<%= escape_javascript(message) %>', '<%= type %>');
```

Comment:
```suggestion
      ThreeScale.toast('<%= raw(escape_javascript(message)) %>', '<%= type %>');
```

without this, I'm getting `'` escaped incorrectly:

![Screenshot From 2025-05-28 11-20-57](https://github.com/user-attachments/assets/e4692a60-0c66-4a7c-8da0-1cb372800a98)